src/i18n/en/nav.ts

```diff
@@ -29,12 +29,12 @@ export default [
 	{ text: 'Markdown', slug: 'guides/markdown-content', key: 'guides/markdown-content' },
 	{ text: 'Routing', slug: 'core-concepts/routing', key: 'core-concepts/routing' },
 	{ text: 'Static Assets', slug: 'guides/imports', key: 'guides/imports' },
+	{ text: 'Troubleshooting', slug: 'guides/troubleshooting', key: 'guides/troubleshooting' },
```

Comment:
If we’re updating `key`, we need to do the same in `fr`, `ja`, and `zh-CN` nav files. The keys of the object there need to match the `key` here. Maybe best just to _delete_ the existing debugging entries there and have translators then come in and add the new troubleshooting item.

Reply:
The better i think is just commenting those and let translator do the rest ?

Reply:
What direction are we taking here?

If we are deleting the debugging file, I'm happy to delete anything from the other languages' `nav.ts` too, because we don't know how quickly they will see the change.

As for the new, Troubleshooting, what was our proceedure before for when a page was not yet translated? Did we ensure there was an English placeholder even before pages were translated, or was there nothing until a translator added this themselves? Whatever that procedure is, that's what we should do here.

What is not going to break the site?  😅

Reply:
Nothing will break it! If there’s no matching nav entry it will fall back to the English text. For example Portuguese in the current deploy preview:

<img width="255" alt="image" src="https://user-images.githubusercontent.com/357379/171938962-95d75e5f-6a94-46c5-ba74-9aebeb38f375.png">


I’m for deleting the incorrect entries and letting them get replaced as translations come in.